Sources/Brave/Frontend/UserContent/UserScripts/Scripts_Dynamic/Scripts/Paged/RewardsReportingScript.js

```diff
@@ -41,7 +41,7 @@ window.__firefox__.includeOnce("RewardsReporting", function($) {
         this.addEventListener('load', listener, true);
         this.addEventListener('error', listener, true);
         return originalOpen.apply(this, arguments);
-    });
+    }, /*overrideToString=*/false);
```

Comment:
What was the reason to override `XMLHttpRequest.prototype.toString` in the first place?